lang/es_ES.json

```diff
@@ -0,0 +1,313 @@
+{
+    ":count posts in the last 24 hours": ":count publicaciones en las últimas 24 horas",
+    ":count posts in the last 7 days": ":count publicaciones en los últimos 7 días",
+    "Additional Posts": "Publicaciones adicionales",
+    "Forum Index": "Índice del foro",
+    "in": "en",
+    "Last Post By": "Última publicación por",
+    "Message": "Mensaje",
+    "Next :count": "Próximos :count",
+    "Previous :count": "Anteriores :count",
+    "Recent Posts": "Publicaciones recientes",
+    "More": "Más",
+    "Close": "Cerrar",
+    "More pages": "Más páginas",
+    "Subset": "Subset",
+    "Disclaimer about ROMs": "Aviso sobre las ROM",
+    "RetroAchievements.org does not condone or supply any copyright-protected ROMs to be used in conjunction with the supplied emulators.": "RetroAchievements.org no aprueba ni suministra ROM protegidas por derechos de autor para ser utilizadas en conjunto con los emuladores suministrados.",
+    "There are no copyright-protected ROMs available for download on RetroAchievements.org.": "No hay ROM protegidas por derechos de autor disponibles para descargar en RetroAchievements.org.",
+    ":earned of :total achievements unlocked": ":earned de :total logros desbloqueados",
+    ":earned of :total softcore achievements unlocked": ":earned de :total logros softcore desbloqueados",
+    ":earned of :total points earned": ":earned de :total de puntos ganados",
+    ":earned of :total softcore points earned": ":earned de :total de puntos softcore ganados",
+    "Finished": "Cerrado",
+    "Mastered": "Masterizado",
+    "Completed": "Completado",
+    "Beaten": "Superado",
+    "Beaten (softcore)": "Superado (softcore)",
+    ":awardLabel on :awardDate": ":awardLabel en :awardDate",
+    "RetroPoints: A measurement of rarity and estimated difficulty.": "RetroPoints: Una medida de rareza y dificultad estimada.",
+    "Derived from points, number of achievers, and number of players.": "Derivado de puntos, número de logros y número de jugadores.",
```

Comment:
I am wondering if "logros" is a typo. "achievers" in this context represents the number of players who have unlocked the achievement, not the count of achievements.

Reply:
Probably is a typo, I'll check it asap. I think "desbloqueadores" or similar.

Reply:
Humm, the translation of this is "complicated" beacuse "desbloqueadores de logros" is redundant. Achiever is a person that has success, and in this case has unlocked an achievement. But we don't have an appropiate translation for this term...

If I put this into the translator DeepL. It suggests me those translations or alternatives.

![image](https://github.com/user-attachments/assets/8c632c5e-95d2-40df-8a85-ebda7121b846)

I suggest "número de usuarios que han desbloqueado logros" but it's excessive long. I'm really confused about the correct term for this.

Reply:
I have pushed a new translation for this. @wescopeland.